pkg/testcase/cluster.go

```diff
@@ -87,12 +90,18 @@ func TestSonobuoyMixedOS(deleteWorkload bool) {
 	}
 }
 
+// FetchCluster returns the cluster
+func FetchCluster() (*factory.Cluster, error) {
+	cluster := factory.ClusterConfig(GinkgoT())
+	return cluster, nil
+}
+
```

Comment:
Do we want to update other places that are retrieving cluster info to use this function also as part of this? Or should we create a "tech debt" item to do that? I'm fine either way.

Reply:
Yeah keeping minimal change to be within scope. Definitely can be fixed in later PRs (AFAIK its used in selinux tests)